The download step tries to check out origin/master instead of origin/main

Executing the task `download-p5` fails because it tries to check out branch `origin/master`. But the branch is called `origin/main`:

```
npm run download-p5

> p5.ts@1.0.0 download-p5 C:\Test\p5.ts
> node src/cli.js download-p5

Cloning https://github.com/processing/p5.js.git to C:/Test/p5.ts/p5
Cloned repo
Setting branch to master
[Error: no reference found for shorthand 'refs/remotes/origin/master'] {
  errno: -3,
  errorFunction: 'Reference.dwim'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! p5.ts@1.0.0 download-p5: `node src/cli.js download-p5`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the p5.ts@1.0.0 download-p5 script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Nobody\AppData\Roaming\npm-cache\_logs\2020-11-30T15_43_50_565Z-debug.log
```

Changing it in src\commands\download-p5.js fixed it:

```
branch: {
    type: 'string',
    default: 'master
    describe: 'branch to check out and update from'
  },
```

to 

```
branch: {
    type: 'string',
    default: 'main',
    describe: 'branch to check out and update from'
  },
```

--- a/src/commands/download-p5.js
+++ b/src/commands/download-p5.js
@@ -19,7 +19,7 @@ exports.builder = {
   },
   branch: {
     type: 'string',
-    default: 'master',
+    default: 'main',
     describe: 'branch to check out and update from'
   },
   silent: {

--- a/src/git.js
+++ b/src/git.js
@@ -66,7 +66,7 @@ function _getRepositoryLatest(url, opts) {
  */
 function getRepositoryLatest(url, opts) {
   if (typeof opts === 'string') opts = { path: opts };
-  if (!opts.branch) opts.branch = 'master';
+  if (!opts.branch) opts.branch = 'main';
 
   return _getRepositoryLatest(url, opts);
 }

